src/aleph_client/conf.py

```diff
@@ -38,6 +42,21 @@ class Config:
         case_sensitive = False
         env_file = ".env"
 
-
 # Settings singleton
 settings = Settings()
+
+if settings.CONFIG_HOME is None:
```

Comment:
Should this not run when the settings are initialised instead of in the main flow of the module ? 
Maybe something like https://github.com/aleph-im/aleph-vm/blob/main/vm_supervisor/conf.py#L159 ?

Reply:
Yes, it will immediately run when you import it. Therefore settings.CONFIG_HOME will always be None

Reply:
Unless the environment variable `ALEPH_CONFIG_HOME` specifies otherwise, since Pydantic will load the environment variables on initialisation.